lib/plugins/blocks.js

```diff
@@ -423,7 +423,7 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
     }
   })
 
-  bot.updateSign = (block, text) => {
+  bot.updateSign = (block, text, isFrontText = true) => {
```

Comment:
Instead of returning a boolean here you can return an integer for side (0 = main, 1 for alternate, and any future side)

Reply:
Let's just flip the condition for now, that way it can be converted to numbers later on if needed